### ISSUE TITLE
Docker Hub stopped publishing with v0.2.6

### ISSUE THREAD

**Issue by grokify** [grokify/ringcentral-permahooks#196]
Latest release is v0.2.9 on July 27, 2024

Last version is v0.2.6 on Docker Hub from Sept 4, 2023.

https://hub.docker.com/r/grokify/ringcentral-permahooks

**Comment by grokify** [comment]
Docker Hub indicates last usage on July 27.

There may be an issue with execution.

**Comment by grokify** [comment]
Failed here:

* v0.2.9: https://github.com/grokify/ringcentral-permahooks/actions/runs/10129063100
* v0.2.8: https://github.com/grokify/ringcentral-permahooks/actions/runs/9056634461
* v0.2.7: https://github.com/grokify/ringcentral-permahooks/actions/runs/6753009597

Succeeded:

* v0.2.6: https://github.com/grokify/ringcentral-permahooks/actions/runs/6072624485

**Comment by grokify** [comment]
Related issue:

https://github.com/docker/build-push-action/issues/984

**Comment by grokify** [comment]
After adjusting the `go.mod` file: in 49f71a441b95faef853a6c1b811ad66922dedcd4

Now have the following error regarding the `cmp`, `map`, and `slices` packages introduced in Go 1.21.

https://github.com/grokify/ringcentral-permahooks/actions/runs/10750645552

```
/go/pkg/mod/go.opentelemetry.io/otel@v1.29.0/attribute/set.go:7:2: package cmp is not in GOROOT (/usr/local/go/src/cmp)
/go/pkg/mod/google.golang.org/grpc@v1.66.0/experimental/stats/metricregistry.go:22:2: package maps is not in GOROOT (/usr/local/go/src/maps)
/go/pkg/mod/google.golang.org/grpc@v1.66.0/clientconn.go:27:2: package slices is not in GOROOT (/usr/local/go/src/slices)
```

**Comment by grokify** [comment]
Issue is here is that the tes is running Go 1.18, not Go 1.21, 1.22, or 1.23.

```
#9 [1/7] FROM docker.io/library/golang:1.18-alpine@sha256:77f25981bd57e60a510165f3be89c901aec90453fd0f1c5a45691f6cb1528807
#9 resolve docker.io/library/golang:1.18-alpine@sha256:77f25981bd57e60a510165f3be89c901aec90453fd0f1c5a45691f6cb1528807 done
```

Ref: https://github.com/grokify/ringcentral-permahooks/actions/runs/10750645552/job/29817057139#step:5:216

**Comment by grokify** [comment]
Fixed in 1f0625eb5f04c0f1b8f59e53ba001e88d6fe76f7
